index.html

```diff
@@ -471,6 +471,14 @@ <h2>User Interface Language</h2>
         </p>
       </section>
     </section>
+    <section>
+      <h2 id="privacy">Privacy Considerations</h2>
+      <p>
+        By creating a new property on the header and <pre class="js">navigator</pre> object this 
```

Comment:
I'd like this text to reflect that there are ways of implementing GPC that pose no fingerprinting risk.  For example, either of the following would be ways of implementing GPC w/o increasing FP risk:

1. have it enabled for all users (e.g., Brave)
2. have it enabled in parallel with another, existing fingerprinting signal (e.g., always enable GPC whenever resistFingerprinting is enabled in Firefox)

I suggest the following text instead:

> Creating a new property on the header and navigator object may create a small increase in fingerprinting risk.  Implementors can mitigate or remove this fingerprinting risk (e.g., enabling for all users if appropriate, or enabling as part of a larger already-page-detectable set of privacy features), and MAY make efforts to do so.

Reply:
I think that risks engaging with what was perceived as problematic in DNT.  That is, if the intent was an explicit opt-out, even the implication that this signal might be sent by default invites an interpretation of the signal as meaningless.

I think that you don't need to shy away from this so much.  Just be up front about it.  Users are partitioned into two groups in a way using this signal.  If this signal is orthogonal to other fingerprinting signals, that further reduces the size of any anonymity set into which users fall when combined with those other fingerprinting signals.  However, if those signals are aligned with other observable changes (like enabling other privacy features) the net effect on anonymity set size might be negligible, because some - or possibly all - of the newly-created partitions would be empty as a result.

You certainly don't need to say "we've tried really hard not to increase fingerprinting", though the effort is definitely appreciated.  This might be a signal that is worth spending a bit on.

Reply:
Just noting quickly here that GPC isn't in the DNT situation. The law for GPC was created before the signal and defines its conditions of application (at least in jurisdictions already confirmed to be applicable). New jurisdictions could of course create new problems, but such is the difficulty in working across governance domains.

Reply:
Agree with @martinthomson. I don't think there's need to give an apology. Yet, it might be worth pointing out, instead, that there's a responsbility on the makers of browsers to alert their users to the possiblity of setting the GPC flag, which would reduce fingerprinting risks (and even make the signal more meaningful if users are properly asked).

Reply:
> that there's a responsbility on the makers of browsers to alert their users to the possiblity of setting the GPC flag

no, this is exactly what we need to avoid in the text.  If the text ends up suggesting "browsers should warn users that enabling the flag will a small increase in fingerprinting", that is a bad outcome.  We should also avoid phrasings that suggest some forms of the signal are more "meaningful" than others. GPC is fine to be on by default, the signal is just as meaningful when its on by default, and GPC on by default removes the fingerprinting risk.

I'm fine if folks want to say "depending on how browsers integrate GPC, it might create a fingerprinting risk", I wont fight about it. But I dont agree with any text that suggests 1. GPC == fingerprinting risk, or 2. asking is better than on by default